extension.Rmd

```diff
@@ -0,0 +1,175 @@
+
+```{r}
+library(separationplot)
+library(Hmisc) 
+
+model_1 <- glm(ext_vote ~ male + age_1 + age_2 + age_4 + edu_1 + edu_2 + farmer_own + worker + retired + unemployed + partner + left_right_scale + country - 1, data = data)
+summary(model_1)
+actual <- model_1$y # Create vector with actual values for vote intention for extreme right parties
+
+log.odds <- model_1$fitted.values # Create a vector with log-odd values
+# Create link function to transform log-odds into predicted probabilities
+link <- function(P) {
+  1/(1+exp(-P))
+} 
+pred <- link(log.odds) # Transform log-odds into predicted probabilities
+pred <- as.vector(pred) 
+range(pred) # variation in the predicted probabilities: very little variation.
+dev.off()
+# Create separation plot to visualize the predictive power of the model
+separationplot(pred = pred, actual = actual, type = "rect", heading = "Model1: The Adorno Thesis", xlab = "ER Vote Intention", newplot = F, line = T, lwd2 = 0.8, height = 0.5, show.expected = T) 
+```
+
+![Model 1](figures/SP.Model1.png)
+
+#Conclusion
+There isn't a big range in the predicted probabilities of the model, therefore the predicted probabilities are not very helpful. Separation plot helps a lot to discern the predictive power of the model. As the events of extreme right vote intension increases as we move to the right (i.e. red lines cluster around the end of the graph), the model does a good job of estimating the likelihood of events. However, as the small arrow in the graph shows, the model predicts that half the number of observations will have the intention to vote for an extreme right party. The number of actual events (red lines) is much lower.  
+
+```
+
+# Model 2: Parsons Thesis (Social Disintegration)
+
+Parsons' 1942 theory of social disintegration posits that anomia, a breakdown of social norms, and attendant feelings of anxiety, anger and isolation brings about a longing for strong leadership and rigid ideologies that extreme right parties can provide.  While we do not have measures for the emotions of participants in the studies whose data we use, in our second model we examine the relationship between extreme right voting and the kinds of sentiments about social change that Parsons refers to: satisfaction with democracy and the negative evaluation of the European Union.
+
+
+```{r}
+model_2 <- glm(ext_vote ~ eu_neg + z_dem_satis + country - 1, data = data)
+summary(model_2)
+
+actual <- model_2$y # Create vector with actual values for vote intention for extreme right parties
+
+log.odds <- model_2$fitted.values # Create a function with log-odds
+# Create link function to transform log-odds into predicted probabilities
+link <- function(P) {
+  1/(1+exp(-P))
+} 
+pred <- link(log.odds) # Transform log-odds into predicted probabilities
+pred <- as.vector(pred) 
+range(pred) # variation in the predicted probabilities: very little variation.
+dev.off()
+# Create separation plot to visualize the predictive power of the model
```

Comment:
this dev.off here makes me think that this code was copy and pasted.  Is that right?